Tests/Functional/phpunit.xml

```diff
@@ -0,0 +1,21 @@
+<phpunit
+    backupGlobals="true"
+    backupStaticAttributes="false"
+    bootstrap="../../.Build/vendor/nimut/testing-framework/res/Configuration/FunctionalTestsBootstrap.php"
+    colors="true"
+    convertErrorsToExceptions="true"
+    convertWarningsToExceptions="true"
+    forceCoversAnnotation="false"
+    processIsolation="true"
+    stopOnError="false"
+    stopOnFailure="false"
+    stopOnIncomplete="false"
+    stopOnSkipped="false"
+    verbose="false"
+>
+    <testsuites>
+        <testsuite name="Functional">
+            <directory>./</directory>
+        </testsuite>
+    </testsuites>
+</phpunit>
```

Comment:
Can't we use the configuration files from the nimut testing framework and provide the paths to the unit/functional tests on the command line?